meinberlin/apps/contrib/cache_helpers.py

```diff
@@ -22,14 +23,19 @@ def get_key(namespace: str, suffix: str) -> str:
     return "_".join(terms)
 
 
-def cache_delete(namespace: str, context: dict[str, str] = None):
+def cache_delete(namespace: str, context: Dict[str, str] = None):
     if REDIS_IS_ENABLED:
         keys = REDIS_CLIENT.keys(pattern=f"*{namespace}*")
         REDIS_CLIENT.delete(names=keys)
 
         logger.info(f"cache delete: {namespace=}, {len(keys)=}, {context=}")
 
 
+@shared_task
+def cache_delete_task(namespace: str):
+    cache_delete(namespace=namespace, context=dict(caller="celery"))
+
+
```

Comment:
tasks should be in a tasks.py file

Now the reason I asked for this task to be in a separate PR is because it needs scheduling with celerybeat. You can start off in a new PR and we can take over if you don't have time to finish by next week.
https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html
@hklarner

Reply:
ok removed it from this PR.